src/puppetlabs/trapperkeeper/authorization/ring.clj

```diff
@@ -16,16 +16,16 @@
 
 ;; Functions
 
-(schema/defn authorized-authentic? :- schema/Bool
-  "Get whether the authorized client is considered authentic or not."
+(schema/defn authorized-authenticated :- schema/Bool
+  "Get whether the authorized client is considered authenticated or not."
   [request :- Request]
-  (get-in request [:authorization :authentic?]))
+  (get-in request [:authorization :authenticated]))
 
-(schema/defn set-authorized-authentic? :- Request
-  "Set whether the authorized client is considered authentic or not."
+(schema/defn set-authorized-authenticated? :- Request
```

Comment:
Since these functions are supposed to be symmetrical and we got rid of the `?` everywhere else, we should remove it from this function name as well.

:metal: